pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [tool.poetry]
-name = "turbo-django"
+name = "turbo"
```

Comment:
Don't you want to keep `turbo-django` ? `turbo` seems to refer to another package https://pypi.org/project/turbo/

Reply:
Playing around with this now - poetry seems to require the `src/` directory has the same value as the [tool.poetry:name] value. And I'd prefer to keep the simplicity of `import turbo`.

Reply:
It looks like the package name in that file is only used to refer to where the source directory resides.  It won't interfere with that project.  And as that looks like a web framework that hasn't been updated in 3 years, it seems unlikely that any user is going to be using that in the same environment as this library.

Reply:
Saw this come through my email and thought I'd just lob in that with poetry, you should still be able to have your src directory (and `import turbo`) named differently from your pypi package `name`: https://python-poetry.org/docs/pyproject/#packages

Used it in a similar way myself for a Django app recently: https://github.com/dropseed/django-admincharts/blob/0b5952fedbf7e4712f18f6c30c3fa81e32d2cc21/pyproject.toml#L12-L14

Reply:
Thanks - looking for exactly that.